SchedulerJobs/SchedulerJobs.Services.UnitTests/AnonymiseHearingsConferencesDataServiceTests.cs

```diff
@@ -43,6 +43,8 @@ public void Setup()
             var userProfile = new UserProfile { UserRole = "Individual" };
             _userApiClient.Setup(x => x.GetUserByAdUserNameAsync(It.IsAny<string>())).ReturnsAsync(userProfile);
 
+            _logger.Setup(x=> x.IsEnabled(LogLevel.Information)).Returns(true);
```

Comment:
rather than doing this in each test, this could be done as part of the `Setup` method, making the tests cleaner:

```
_logger.Setup(x=> x.IsEnabled(LogLevel.Error)).Returns(true);
_logger.Setup(x=> x.IsEnabled(LogLevel.Warning)).Returns(true);
 _logger.Setup(x=> x.IsEnabled(LogLevel.Information)).Returns(true);
```

same for the other test files

Reply:
moved

Reply:
can you do this for the other tests too?

Reply:
done